### DISCUSSION TITLE
practice-exam/practice-exam-20

### DISCUSSION THREAD

**Discussion by giscus** [kananinirav/AWS-Certified-Cloud-Practitioner-Notes#242] [Comments]
# practice-exam/practice-exam-20

If you are preparing for AWS Cloud Practitioner Exam, this guide will help you with quick revision before the exam. I share the notes I used to study and pass exam.

https://kananinirav.com/practice-exam/practice-exam-20.html

**Comment by sendurangr** [comment]
34. Who is responsible for patching the guest operating system for Amazon RDS?
**A. The AWS Product team**
B. The customer Database Administrator
C. Managed partners
D. AWS Support

isn't it not A. The AWS Product team

**Reply by suhailSallam** [reply]
This is the answer of DeepSeek:

The correct answer is: B. The customer Database Administrator

Explanation:
For Amazon RDS (Relational Database Service), AWS manages the underlying infrastructure, including the host OS, database engine updates, and security patches for the RDS service itself. However, the customer is responsible for patching the guest operating system (the OS layer within the database instance) if applicable, as well as managing database settings, user access, and application-level security.

**Reply by emilian** [reply]
For standard Amazon RDS, AWS is responsible for patching the operating system and database software.

**Comment by suhailSallam** [comment]
28. A web application is hosted on AWS using an Elastic Load Balancer, multiple Amazon EC2 instances, and Amazon RDS.
Which security measures fall under the responsibility of AWS? (Choose two.)
A. Running a virus scan on EC2 instances
B. Protecting against IP spoofing and packet sniffing
C. Installing the latest security patches on the RDS instance
D. Encrypting communication between the EC2 instances and the Elastic Load Balancer
E. Configuring a security group and a network access control list (NACL) for EC2

Correct answers are: B & D not B & C.

**Reply by ghost** [reply]
The security measures that fall under the responsibility of AWS in this scenario are:
B. Protecting against IP spoofing and packet sniffing: AWS is responsible for the security of the underlying infrastructure, including protecting against network-level attacks like IP spoofing and packet sniffing within their network.
C. Installing the latest security patches on the RDS instance: Amazon RDS is a managed service, meaning AWS is responsible for managing the underlying infrastructure, including applying security patches to the database engine and operating system of the RDS instance.
